oss_src/unity/python/sframe/test/test_sarray.py

```diff
@@ -1674,3 +1674,15 @@ def test_save_sarray(self):
         data.save(tmp_dir)
         shutil.rmtree(tmp_dir)
         print data
+
+    def test_to_numpy(self):
+        X = SArray(range(100))
```

Comment:
Is there limitation on the types that supported by SFrame but not by Numpy?

Also would be nice to have 'missing value' test too.